src/crab/ebpf_domain.hpp

```
@@ -19,8 +21,8 @@ namespace prevail {
 // the offsets get replaced with 64-bit address pointers.  However, we currently
 // need to do pointer arithmetic on 64-bit numbers so for now we cap the interval
 // to 32 bits.
-constexpr int MAX_PACKET_SIZE = 0xffff;
-constexpr int64_t PTR_MAX = std::numeric_limits<int32_t>::max() - MAX_PACKET_SIZE;
+inline int max_packet_size() noexcept { return thread_local_options.max_packet_size; }
```

Clang (14.0.6)

[warning] use a trailing return type for this function

(modernize-use-trailing-return-type)